filebeat/beater/channels.go

```diff
@@ -5,46 +5,43 @@ import (
 	"sync/atomic"
 
 	"github.com/elastic/beats/filebeat/input"
-	"github.com/elastic/beats/filebeat/registrar"
-	"github.com/elastic/beats/filebeat/spooler"
 )
 
-type spoolerOutlet struct {
-	wg      *sync.WaitGroup
-	done    <-chan struct{}
-	spooler *spooler.Spooler
-
+type prospectorOutput struct {
+	wg     *sync.WaitGroup
+	done   <-chan struct{}
+	input  chan *input.Event
 	isOpen int32 // atomic indicator
 }
 
-type publisherChannel struct {
+type spoolerOutput struct {
 	done chan struct{}
 	ch   chan []*input.Event
 }
 
-type registrarLogger struct {
+type publisherOutput struct {
 	done chan struct{}
 	ch   chan<- []*input.Event
 }
 
-type finishedLogger struct {
+type logger struct {
```

Comment:
while `finishedLogger` is a bad naming, I wouldn't call it `logger` either, as logger somehow implies this is about logging a message to file.